<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_employee</class>
 <widget class="QMainWindow" name="NewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>code:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="code">
      <property name="objectName">
       <string>code</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>first name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="first_name">
      <property name="objectName">
       <string>first_name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>last name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="last_name">
      <property name="objectName">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_button">
      <property name="text">
       <string>Add employee</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>